md/architecture.md

```diff
@@ -1,7 +1,7 @@
 # Architecture
 
 Bonita Intelligent Continuous Improvement use data Bonita engine database and store them in an Elasticsearch storage
```

Comment:
Bonita Intelligent Continuous Improvement Add-on (ICI Add-on) uses data from the Bonita Engine database and stores